apps/frontend/src/pages/servers/manage/[id].vue

```diff
@@ -774,6 +791,23 @@ onMounted(() => {
     connectWebSocket();
   }
 
+  Intercom({
+    app_id: INTERCOM_APP_ID.value,
+    userId: userId.value,
+    name: username.value,
+    email: email.value,
+    created_at: createdAt.value ?? undefined,
+  });
+
+  // duplication needed because we shutdown intercom in cleanup
```

Comment:
This should NOT be done. This is no good, very bad!!! You are basically telling icom to init more then once and this CAN creates unspeakable horrors. You should either call Intercom with auto boot set to true, or boot().

Reply:
boot() without initializing via Intercom() causes errors, and autoboot doesnt exist